### PR TITLE
User can give a title to the Plot

### PR DESCRIPTION
**Pull request by MariaAdner**

### CODE REVIEW

File: Components/GraphTitle/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  function handleTitle() {
+    if (titleLabel.length > 0) {
+      alert("Title of the graph is assigned");
+    }
+  }
+
+  function completedTitle() {
+    return titleLabel.length === 0;
+  }
```
Comment (MerleMak) [inline Components/GraphTitle/index.js:25]: you can handle this directly in the argument of the InputTypeSubmit :)

File: Components/GraphTitle/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                idString={"Title"}
+                placeholderString={"Fill me with a title"}
+                onChange={onTitleChange}
+              />
+              <br></br>
+              <br></br>
+              <StyledInputTypeSubmit
+                value="Next"
+                onClick={handleTitle}
+                disabled={completedTitle()}
```
Comment (MerleMak) [inline Components/GraphTitle/index.js:46]:
```suggestion
                disabled={!titleLabel.length}
```

**Review by MerleMak — COMMENTED** [review]
very nice job!!!

Keep it up, The app is getting better and better 🤩

File: Components/GraphTitle/index.js (hunk 1 above)
Comment (MariaAdner) [reply Components/GraphTitle/index.js:25]: We wanted to be consistent with the other components. So we keep it like this.

File: Components/GraphTitle/index.js (hunk 2 above)
Comment (MariaAdner) [reply Components/GraphTitle/index.js:46]: See the comment above.

**Pull request merged**